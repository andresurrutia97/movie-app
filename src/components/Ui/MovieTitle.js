import React from 'react';
import PropTypes from 'prop-types';

const MovieTitle = ({ title }) => (
  <h2 className="font-bold text-lg md:text-4xl lg:text-5xl">{title}</h2>
);

MovieTitle.propTypes = {
  title: PropTypes.string,
};

MovieTitle.defaultProps = {
  title: '',
};

export default MovieTitle;
