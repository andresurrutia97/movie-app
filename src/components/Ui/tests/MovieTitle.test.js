import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import MovieTitle from '../MovieTitle';

describe('<MovieTitle/>', () => {
  const component = <MovieTitle title="Test" />;

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a text equal to "Test"', () => {
    const wrapper = shallow(component);
    expect(wrapper.props().children).toBe('Test');
  });
});
