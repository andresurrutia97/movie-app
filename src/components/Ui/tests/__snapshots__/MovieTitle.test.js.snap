// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieTitle/> should render correctly  1`] = `
<h2
  className="font-bold text-lg md:text-4xl lg:text-5xl"
>
  Test
</h2>
`;
